src/shred_network/repair_service.zig

```diff
@@ -110,14 +123,17 @@ pub const RepairService = struct {
         self.report.deinit();
     }
 
-    const min_loop_duration_ns = 100 * std.time.ns_per_ms;
+    const min_loop_duration_ns = 500 * std.time.ns_per_ms;
```

Comment:
sig Durations instead of `_ns` ?

Reply:
This wasn't even used, I deleted it here c323f03d45f332b533ef2a96d7ba81144f6325b4